File: widgetssdk/src/main/java/com/glia/widgets/core/notification/device/INotificationManager.java
package com.glia.widgets.core.notification.device;

public interface INotificationManager {
    void showAudioCallNotification();

    void showVideoCallNotification(boolean isCallVisualizer);

    void removeCallNotification();

    void showScreenSharingNotification();

    void removeScreenSharingNotification();
}

File: widgetssdk/src/main/java/com/glia/widgets/core/notification/device/INotificationManager.kt
package com.glia.widgets.core.notification.device

internal interface INotificationManager {
    fun showAudioCallNotification()
    fun showVideoCallNotification(isTwoWayVideo: Boolean, hasAudio: Boolean)
    fun removeCallNotification()
    fun showScreenSharingNotification()
    fun removeScreenSharingNotification()
    fun startNotificationRemovalService()
}
